### PR TITLE
Teaming/user management

### PR DESCRIPTION
**Pull request by vanshBhatia-A4k9**
**What this PR does / why we need it**:

 - Added Concepts for `Teaming` & `User Management`
 - Added User Guides for `Teaming` & `User Management`
 - Added asset level structure
 - Improved teaming/user management sidebar flow.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [x] Signed the commit for DCO to be passed
-   [x] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Review by arkajyotiMukherjee — APPROVED** [review]

File: website/docs/concepts/teaming.md [hunk 1]
```diff
@@ -4,4 +4,36 @@ title: Collaborate with Teams
 sidebar_label: Teaming
 ---
 
-## Coming soon
+The litmus portal has a built in teaming feature to facilitate collaboration between users using project level role access, the core concepts of which are discussed below.
```
Comment (Jonsy13) [inline website/docs/concepts/teaming.md]: It should be Litmus Portal or Chaos Center? @arkajyotiMukherjee

**Review by Jonsy13 — APPROVED** [review]

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]
The main content is well written. Thanks, team.
Can you also provide the other sections (like Learn More, Resources, Pre-requisites, Before you begin etc) wherever applicable to the Concepts and User guides
https://docs.google.com/document/d/10TjEEZgFgYmYBjyBT2IF3TlZcvEn3aVH6Jqm47scxbA/edit#heading=h.2odiu0a6pc21

File: website/docs/concepts/teaming.md (hunk 1 above)
Comment (SarthakJain26) [inline website/docs/concepts/teaming.md]: Can we update litmus portal to chaos center? 
Cc: @arkajyotiMukherjee

Comment (arkajyotiMukherjee) [reply website/docs/concepts/teaming.md]: Yes, Litmus Portal was the internal name...we should update it to `ChaosCenter`

Comment (arkajyotiMukherjee) [reply website/docs/concepts/teaming.md]: `ChaosCenter` with no spaces is the correct term. Thanks for pointing it out.

File: website/docs/concepts/user-management.md [hunk 2]
```diff
@@ -4,4 +4,20 @@ title: User Management
...
+### Portal level user Roles (Admin, Non-Admins)
+ChaosCenter supports two portal level roles for defining the privilege levels of a certain user:
+
+**Admin:** the admin user is created by default on initial project setup and can log into the portal using the credentials `admin/litmus` once the server pod (auth container) is up.
+
+**Non-admin users:** The admin of the portal has the exclusive ability to create any number of non-admin users.
+
+### Role Privileges
+
+**Admin** is the highest privilage level offered in the portal and the admin has complete access to all the features offered by the portal.
```
Comment (Saranya-jena) [inline website/docs/concepts/user-management.md]: `privilage` -> `privilege`

File: website/docs/concepts/teaming.md [hunk 3]
```diff
@@ -4,4 +4,36 @@ title: Collaborate with Teams
...
+- Remove a member from your project.
+- Check the status of the invitation you sent to other members.
+- Edit the user role in case the invitation is pending.
+- Cancel the invitation.
+
+***As a Viewer or Editor you can:***
+
+- Check and Accept/Decline the received invitations
+- Switch and browse any project you are a member of.
+- Can leave the Project you are a part of, except your own.
```
Comment (Saranya-jena) [inline website/docs/concepts/teaming.md]: `Project` -> `project`

File: website/docs/concepts/teaming.md [hunk 4]
```diff
@@ -4,4 +4,36 @@ title: Collaborate with Teams
...
+- View the list of team members with other details including their role in the project, email-id, date-time of joining the project team.
+- Rename your project.
+- Remove a member from your project.
+- Check the status of the invitation you sent to other members.
+- Edit the user role in case the invitation is pending.
+- Cancel the invitation.
+
+***As a Viewer or Editor you can:***
+
+- Check and Accept/Decline the received invitations
```
Comment (Saranya-jena) [inline website/docs/concepts/teaming.md]: Add a full stop

File: website/docs/user-guides/accept-invite.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+id: accept-invite
+title: Accept an invite
+sidebar_label: Accept invite
+---
+
+Once the invitation is received, the user can take a look at details including the sender's name, role, and project name, then decide whether to accept or decline the invitation.
```
Comment (Saranya-jena) [inline website/docs/user-guides/accept-invite.md]: the user -> you

File: website/docs/user-guides/create-user.md [hunk 6]
```diff
@@ -4,4 +4,27 @@ title: Create a User
...
 
-## Coming soon
+
+This feature enables the admin to create a new user by assigning a unique username and password for the user. In addition to this, the admin can also provide the name and email address of the new user which is optional. 
+
+## Steps
+
+### 1. Create a new user
+
+click the `Create new user` button as observed below:
```
Comment (Saranya-jena) [inline website/docs/user-guides/create-user.md]: click -> Click on

File: website/docs/user-guides/deactivate-user.md [hunk 7]
```diff
@@ -4,4 +4,24 @@ title: Deactivate a User
...
+
+### 1. Locate and deactivate the user
+
+In the user management tab, locate the user account that you'd like to deactivate and click on the horizontal options icon.
+
+<img src={require('../assets/user-guides/user-management/deactivate-user/step-1.png').default} width="1000" height="500" />
+
+### 2. Confirm the user is deactivated
+
+Once deactivated, the indicator next to the users username in the usermanagement table will turn gray.
```
Comment (Saranya-jena) [inline website/docs/user-guides/deactivate-user.md]: users -> user's

File: website/docs/user-guides/edit-invite.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+---
+id: edit-invite
+title: Edit or Cancel an invite
+sidebar_label: Edit/Cancel invite
+---
+
+If you had a change of mind and you wanted to change the role of an invitation that has been already sent, we got you!
+
+Just go to the invited tab, change the role, and hit the `Resend` button. You can also cancel the invitation by just clicking on the Bin icon.
```
Comment (Saranya-jena) [inline website/docs/user-guides/edit-invite.md]: Bin -> `bin`

File: website/docs/user-guides/invite-team-member.md [hunk 9]
```diff
@@ -4,4 +4,25 @@ title: Invite a Team Member
 sidebar_label: Invite Team Member
 ---
 
-## Coming soon
+
+With this feature, you can select as many users you want, choose their roles individually and send the invitation at once! Once it is done successfully you can see the status of the sent invitation (whether it is in a pending/accepted/declined or exited state) along with all the other necessary details in the Invited tab.
```
Comment (Saranya-jena) [inline website/docs/user-guides/invite-team-member.md]: Invited -> `Invited`

File: website/docs/user-guides/remove-team-member.md [hunk 10]
```diff
@@ -4,4 +4,18 @@ title: Remove a Team Member
...
 ---
 
-## Coming soon
+If you are the project owner, you have the ability to remove any members from your project as required.
+
+## Steps
+
+### 1. Identify the member to remove
+
+In the `Team` tab in the settings page, scroll down to the `My project` table, here you will be able to see all the members who have their invitation staus as `accepted` for your project collaboration invite. Here, identify the user to remove and click on the red `Remove` bin icon as shown.
```
Comment (Saranya-jena) [inline website/docs/user-guides/remove-team-member.md]: staus -> status

**Review by Saranya-jena — CHANGES_REQUESTED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**